benchmark/stargz_utils.go

```diff
@@ -16,13 +16,104 @@
 
 package benchmark
 
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+	"time"
+)
+
+var (
+	stargzAddress = "/tmp/containerd-stargz-grpc/containerd-stargz-grpc.sock"
+	stargzRoot    = "/tmp/lib/containerd-stargz-grpc"
+	stargzConfig  = "../snapshotter_config.toml"
```

Comment:
the snapshotter config reads
```
[cri_keychain]
  enable_keychain = true 
  image_service_path = "/tmp/containerd-grpc/containerd.sock"
```
I'm guessing we can use it for both?

Reply:
yes